### ISSUE TITLE
The testset always can not be reconstructed

### ISSUE THREAD

**Issue by imsleepy711** [Justin-Tan/generative-compression#38]
Hi Justin, thanks for your codes. Here is my question. 
I used another train set for training, about 1400 pictures. And I set the batch size to 1, epochs to 400, the number of steps is close to 21w. I found that the generator loss is stable around 40. Then I stopped and tested. The most of the train set have the good reconstruction effect, but most of the test set have no good result, like this photo. I tried to train a few times and got the same result——the test set can not be reconstructed. Please do you have any advice? Thank you!
![buildings06_compressed](https://user-images.githubusercontent.com/61038243/94565097-54c03080-029b-11eb-84bc-24f4ae7a5f17.png)

**Comment by Justin-Tan** [comment]
The training set is probably too small - you can either try finetuning from a pretrained model on a larger dataset or use the approach in this newer repo: https://github.com/Justin-Tan/high-fidelity-generative-compression - finetuning on the pretrained models there should work well. Note that the bitrate for the 'high-fidelity' models is generally higher than the models in this repo, but with better reconstruction quality.

**Comment by imsleepy711** [comment]
Thank you! I will try your advice.
